sdk/monitor/monitor-query/swagger/metric-definitions.md

```diff
@@ -22,4 +22,6 @@ api-version-parameter: choice
 hide-clients: true
 v3: true
 typescript: true
+use-extension:
+  "@autorest/typescript": "6.0.0-beta.15"
```

Comment:
should we be using the latest release of `6.0.13`? Or is there a reason we need to use an old beta?

Reply:
Discussed offline